.rubocop.yml

```diff
@@ -1,341 +1,9 @@
-require:
-  - rubocop-packaging
-  - rubocop-performance
-  - rubocop-rails
-
-AllCops:
-  TargetRubyVersion: 3.0
-  # RuboCop has a bunch of cops enabled by default. This setting tells RuboCop
-  # to ignore them, so only the ones explicitly set in this file are enabled.
-  DisabledByDefault: false
-  Exclude:
-    - '**/tmp/**/*'
-    - '**/templates/**/*'
-    - '**/vendor/**/*'
-    - 'actionpack/lib/action_dispatch/journey/parser.rb'
-    - 'actionmailbox/test/dummy/**/*'
-    - 'actiontext/test/dummy/**/*'
-    - '**/node_modules/**/*'
-
-Performance:
-  Exclude:
-    - '**/test/**/*'
-
-# Prefer assert_not over assert !
-Rails/AssertNot:
-  Include:
-    - '**/test/**/*'
-
-# Prefer assert_not_x over refute_x
-Rails/RefuteMethods:
-  Include:
-    - '**/test/**/*'
-
-Rails/IndexBy:
-  Enabled: true
-
-Rails/IndexWith:
-  Enabled: true
-
-# Prefer &&/|| over and/or.
-Style/AndOr:
-  Enabled: true
-
-Layout/ArgumentAlignment:
-  Enabled: true
-
-Layout/ArrayAlignment:
-  Enabled: true
-
-Layout/BlockAlignment:
-  Enabled: true
-
-# Align `when` with `case`.
-Layout/CaseIndentation:
-  Enabled: true
-
-Layout/ClosingHeredocIndentation:
-  Enabled: true
-
-# Align comments with method definitions.
-Layout/CommentIndentation:
-  Enabled: true
-
-Layout/ElseAlignment:
-  Enabled: true
-
-# Align `end` with the matching keyword or starting expression except for
-# assignments, where it should be aligned with the LHS.
-Layout/EndAlignment:
-  Enabled: true
-  EnforcedStyleAlignWith: variable
-  AutoCorrect: true
-
-Layout/EmptyLines:
-  Enabled: true
-
-Layout/EmptyLineAfterMagicComment:
-  Enabled: true
-
-Layout/EmptyLinesAroundAccessModifier:
-  Enabled: true
-
-Layout/EmptyLinesAroundBlockBody:
-  Enabled: true
-
-# In a regular class definition, no empty lines around the body.
-Layout/EmptyLinesAroundClassBody:
-  Enabled: true
-
-# In a regular method definition, no empty lines around the body.
-Layout/EmptyLinesAroundMethodBody:
-  Enabled: true
-
-# In a regular module definition, no empty lines around the body.
-Layout/EmptyLinesAroundModuleBody:
-  Enabled: true
-
-Layout/ExtraSpacing:
-  Enabled: true
+inherit_gem:
+  rubocop-shopify: rubocop.yml
 
 Layout/LineLength:
   Max: 80
```

Comment:
Of course, this limit is [highly controversial](https://stackoverflow.com/questions/110928/is-there-a-valid-reason-for-enforcing-a-maximum-width-of-80-characters-in-a-code).

I strongly recommend `80` characters as a **soft limit** and we should include that in our style guide, e.g. in the Wiki. In VSCode you can also display a grey border at 80 chars.

![image](https://github.com/MaMpf-HD/mampf/assets/37160523/10385cb6-4e07-4d58-8d5b-2249e9c6e1d7)

However, in my past dev experience, I remember lines that were slightly longer than 80 chars and I was really upset that a linter forced me to break a statement to two lines. Sometimes this can actually make a statement less readable (given that the line was just a bit above 80 chars and not something like 120 chars or even more).

This is probably why they even got rid of the 80-column warning in the Linux kernel [here](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=bdc48fa11e46f867ea4d75fa59ee87a7f48be144). Instead, they raised the limit to `100` chars which I think is more reasonable as a hard limit.